<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1383</width>
    <height>1005</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="DrawingScreen" name="drawingArea" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>761</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>720</y>
      <width>671</width>
      <height>123</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Vermelho</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderRed">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumberRed">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Azul</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderBlue">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumberBlue">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Verde</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderGreen">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumberGreen">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>TransparÃªncia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderTransparency">
         <property name="maximum">
          <number>1</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumberTransparency">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1383</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuSair"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawingScreen</class>
   <extends>QWidget</extends>
   <header>drawingscreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberRed</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>770</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>781</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberBlue</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>804</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>802</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberGreen</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>839</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>840</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTransparency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberTransparency</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>867</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>868</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>killAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>killAll()</slot>
 </slots>
</ui>
